### PR TITLE
feat: home page added

### PR DESCRIPTION
**Pull request by Manishsharma203**
For preview check the link below:
`https://pqrzux.csb.app/pages/homePage.html`

### CODE REVIEW

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
Add search bar on Homepage. (mandatory)

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
Else, It looks good

**Comment by Manishsharma203** [comment]
> Add search bar on Homepage. (mandatory)

Sure. Thanks for pointing out

**Comment by Manishsharma203** [comment]
> Add search bar on Homepage. (mandatory)

I saw it again. It is under `Good to have (P1)` . Correct me If I m missing something.

File: script.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+var currentCarouselNumber = 0;
+const totalCarouselNumber =
+  document.getElementsByClassName("carousel-img").length;
+let carouselImages = document.getElementsByClassName("carousel-img");
+carouselImages[currentCarouselNumber].style.display = "block";
+function pageNext() {
```
Comment (krgarima) [inline script.js:6]: I suggest you use arrow functions

**Review by krgarima — COMMENTED** [review]
I will suggest use ES6 features wherever possible. Rest is fine.

**Pull request merged**